egs/commonvoice/asr1/local/data_prep.pl

```diff
@@ -46,7 +46,7 @@
   $text =~ tr/a-z/A-Z/;
   print TEXT "$uttId"," ","$text","\n";
   print GNDR "$uttId"," ","$gender","\n";
-  print WAV "$uttId"," sox $db_base/$filepath -t wav -r 16k -b 16 -e signed - |\n";
+  print WAV "$uttId"," ffmpeg -i $db_base/$filepath -f wav -ar 16000 -ab 16 - |\n";
```

Comment:
For me, using ffmpeg is fine but why did you update this? I just want to know the reason.

Reply:
We found that sox doesn't support MP3 format because of some legal terms.

Reply:
The default sox does not support mp3 but it can be done with some plug-in install. `ffmpeg` is a robust solution, but it may not be installed as a default tool. So, could you add an installation check of `ffmpeg` like https://github.com/espnet/espnet/blob/83101a26177500542d1fcff9c22bf7752c2f49ad/egs/librispeech/asr1/local/data_prep.sh#L16-L20?